src/odemis/gui/model/__init__.py

```diff
@@ -187,6 +187,9 @@ def __init__(self, microscope):
         # Indicates whether the microscope is acquiring a high quality image
         self.is_acquiring = model.BooleanVA(False)
 
+        # Indicates whether the microscope is performing an alignment process
+        self.is_aligning = model.BooleanVA(False)
+
```

Comment:
Is it possible to merge that commit with the one renaming the VA to is_preparing?